usmqe/api/base.py

```diff
@@ -32,7 +32,7 @@ def print_req_info(resp):
         LOGGER.debug("request.body:  {}".format(resp.request.body))
         LOGGER.debug("request.headers:  {}".format(resp.request.headers))
         LOGGER.debug("response.cookies: {}".format(resp.cookies))
-        # LOGGER.debug("response.content: {}".format(resp.content))
+        LOGGER.debug("response.content: {}".format(resp.content))
```

Comment:
There's still a lot of data in the response. So I understand why it was commented, not sure if it's good to remove the comment now.